command_builder.test.ts

```diff
@@ -10,7 +10,16 @@ Deno.test("Quick example", async () => {
     assertEquals(line, "bug : abcdef");
   }
 
-  const result = await $`echo hello`
-    .xargs((input) => $`echo ${input} world`.stdout("piped"));
-  assertEquals(result[0].stdout, "hello world\n");
+  {
+    const result = await $`echo hello`
+      .xargs((input) => $`echo ${input} world`)
+      .then((output) => output[0].xargs((input) => $`echo ${input} world2`));
+    assertEquals(await result[0].text(), "hello world world2");
+  }
+  {
+    const result = await $`echo "line1\nline2"`
+      .xargs((input) => $`echo ${input} world`);
+    assertEquals(await result[0].text(), "line1 world");
+    assertEquals(await result[1].text(), "line2 world");
+  }
```

Comment:
Thank you for adding test!
Can you separate each block into a separate Deno.test?

Reply:
done